sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace_hub/workspace_hub_config.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         additional_workspace_storage_accounts: Optional[List[str]] = None,
         default_workspace_resource_group: Optional[str] = None,
     ) -> None:
-         
```

Comment:
Unneeded whitespace change.

Reply:
Edit: Unless this was automatically done by a linter. What that this case for this and the _schedule_operations.py file?

Reply:
reverted

Reply:
it's done by running black before, but after revert and run again this doesn't happened, so keep it not changed